.github/workflows/tests.yml

```diff
@@ -177,3 +172,99 @@ jobs:
           pip check
           stestr run --slowest
         shell: bash
+  tests_macos:
+    runs-on: ${{ matrix.os }}
+    needs: [sdist, lint]
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10"]
+        os: ["macOS-latest"]
+    env:
+      AER_THRUST_BACKEND: OMP
+      QISKIT_TEST_CAPTURE_STREAMS: 1
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-${{ matrix.python-version}}-pip-test-${{ hashFiles('setup.py','requirements-dev.txt','constraints.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.python-version}}-pip-test-
+            ${{ runner.os }}-${{ matrix.python-version}}-pip-
+            ${{ runner.os }}-${{ matrix.python-version}}-
+      - name: Install Deps
+        run: |
+          set -e
+          pip install -U -c constraints.txt -r requirements-dev.txt
+      - name: Install Aer
+        run: python -m pip install -U .
+      - name: Run Tests
+        env:
+          QISKIT_SUPPRESS_PACKAGING_WARNINGS: Y
+        run: |
+          set -e
+          pip check
+          stestr run --slowest
+        shell: bash
+        if: runner.os != 'macOS' || matrix.python-version != '3.8'
+      - name: Run Tests macOS 3.8
+        env:
+          QISKIT_SUPPRESS_PACKAGING_WARNINGS: Y
+          QISKIT_IN_PARALLEL: TRUE
+        run: |
+          set -e
+          pip check
+          stestr run --slowest
+        shell: bash
+        if: runner.os == 'macOS' && matrix.python-version == '3.8'
```

Comment:
The Python 3.8 special parallel handling was introduced in #1064, before Python 3.9 was released, and I'm assuming because of issues with `spawn`?  Given that the previous test runs without the environment variable in Python 3.9, is the special handling still necessary?

Reply:
Yeah, we can remove this, it's actually encoded in terra by default now: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/tools/parallel.py#L62-L79

Reply:
Done in: https://github.com/Qiskit/qiskit-aer/pull/1448/commits/89dab544f87dff70a8677d2757c4445306b09642